doc_admin/AdminManual.adoc

```diff
@@ -1416,6 +1417,24 @@ By default, ALLOWPRINTOPENEDBILL is set to no.
 NOTE: An application restart is required to apply the modified setting.
 
 
+==== 3.1.36 PATIENTPHOTO
+
+The following table shows the default value and the allowed ones:
+
+[cols=",,",options="header",]
+|===
+|Key |Default Value |Valid Values
+|PATIENTPHOTO |DB |DB, filesystem path
+|===
+
+Open Hospital can save profile picture on DB or on filesystem. 
+When it is not specified DB but a non empty string, Open Hospital will search for profile picture in the path specified in PATIENTPHOTO parameter.
```

Comment:
Better wording: If the PATIENTPHOTO parameter is not set to DB and is not blank, Open Hospital searches for the profile picture in the path specified in he PATIENTPHOTO parameter.